src/main/java/Ui.java

```diff
@@ -0,0 +1,76 @@
+/**
+ * Ui handles the interactions with the user.
+ */
+public class Ui {
+
+    /**
+     * Prints the welcome message for users when they run the app.
+     */
+    public void showWelcome() {
+        String logo = " ____        _        \n"
+                + "|  _ \\ _   _| | _____ \n"
+                + "| | | | | | | |/ / _ \\\n"
+                + "| |_| | |_| |   <  __/\n"
+                + "|____/ \\__,_|_|\\_\\___|\n";
+        System.out.println("Hello from\n" + logo);
+        System.out.println("Hello! I'm Duke\n" + "What can I help you with today! :-)");
+    }
+
+    /**
+     * Prints the response after successfully adding a task.
+     * @param task Task that was added.
+     * @param taskListSize Number of tasks in the TaskList.
+     */
+    public void responseToAddTask(Task task, int taskListSize) {
+        System.out.println("Done! One new task:\n" + task.toString() + "\nNow you have "
+                 + taskListSize + ((taskListSize == 1) ? " task" : " tasks") + " in the list");
+    }
+
+    /**
+     * Prints the response after successfully deleting a task.
+     * @param task Task that was deleted.
+     * @param taskListSize Number of tasks in the TaskList.
+     */
+    public void responseToDelete(Task task, int taskListSize) {
+        System.out.println("Noted, I've removed this task:\n" + task.toString()
+                 + "\nNow you have " + taskListSize + ((taskListSize == 1) ? " task" : " tasks") + " in the list");
```

Comment:
I like the small detail of adjusting the grammar based on how many tasks there are! At the same time, this ternary makes the expression slightly hard to read. Perhaps you could consider moving it out of the println.